src/prog_models/utils/containers.py

```diff
@@ -4,24 +4,23 @@
 import numpy as np
 from typing import Union
 
-from ..exceptions import ProgModelTypeError
+from prog_models.exceptions import ProgModelTypeError
 
 
 class DictLikeMatrixWrapper():
     """
     A container that behaves like a dictionary, but is backed by a numpy array, which is itself directly accessable. This is used for model states, inputs, and outputs- and enables efficient matrix operations.
-    
-    Arguments
-    ---------
-    keys: list
-        The keys of the dictionary. e.g., model.states or model.inputs
-    data: dict or numpy array
-        The contained data (e.g., :term:`input`, :term:`state`, :term:`output`). If numpy array should be column vector in same order as keys
     """
-    def __init__(self, keys : list, data : Union[dict, np.array]):
-        if not isinstance(keys, list):
-            keys = list(keys)
 
+    def __init__(self, keys: list, data: Union[dict, np.array]):
+        """ Initializes the container
+
+        Arguments:
```

Comment:
For the sphinx doc generation to work correctly, the arguments for the constructor need to be in the doc comments for the class. You can repeat it here if you feel it helps, but add it back to the doc comment above. 

Also, note that doc comments need to be in numpy or google format for sphinx doc generation to work. 

* Numpy: https://numpydoc.readthedocs.io/en/latest/format.html or https://python.plainenglish.io/how-to-write-numpy-style-docstrings-a092121403ba
* Google: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html